app/src/main/java/org/cru/godtools/ui/tooldetails/ToolDetailsFragment.kt

```diff
@@ -133,7 +133,8 @@ class ToolDetailsFragment() : BasePlatformFragment<ToolDetailsFragmentBinding>(R
 
         activity?.run {
             openToolActivity(code, tool.type, locale, showTips = true)
-            if (!settings.isFeatureDiscovered(FEATURE_TUTORIAL_TIPS)) {
+            if (settings.getFeatureDiscoveredCount("$FEATURE_TUTORIAL_TIPS$code") < 3) {
+                settings.setFeatureDiscovered("$FEATURE_TUTORIAL_TIPS$code")
```

Comment:
I assume that ```settings.getFeatureDiscoveredCount``` returns an Int value for how many times the tutorial tips have been shown for a specific tool. And we decide which tool to check by passing ```$code``` in the function as an argument. Where are those values stored?

Reply:
yes, the `Settings` object has all the feature discovery logic. using Flipper you can look at the underlying SharedPreferences that actually stores the raw data.
<img width="1200" alt="Screen Shot 2020-10-08 at 11 37 03 AM" src="https://user-images.githubusercontent.com/47818/95481144-ada65d80-095a-11eb-8eae-0df7ad29c24d.png">

Reply:
`$code` is the tool code (e.g. `kgp`, `fourlaws`)

Reply:
I was able to get Flipper connected and find the stored raw data, thanks!